pulsar-client-go/pulsar-perf-go/perf-producer.go

```diff
@@ -0,0 +1,151 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/apache/pulsar/pulsar-client-go/pulsar"
+	"github.com/bmizerany/perks/quantile"
+	"github.com/spf13/cobra"
+	"github.com/beefsack/go-rate"
+	"log"
```

Comment:
in pulsar-client-go, we have imported the `logrus` library. For the sake of unification, we can use 
```
log "github.com/apache/pulsar/pulsar-client-go/logutil"
```